### PR TITLE
photo-album-cvr

### PR DESCRIPTION
**Pull request by carmenvramos**

### CODE REVIEW

File: src/components/Photos.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    return (
+      <li key={url}>
+        <Photo url={url}/>
+      </li>
+    );
+  });
+
+  return (
+    <ul>
+      <li>{listOfUrls}</li>
```
Comment (grayson073) [inline src/components/Photos.js]:
```suggestion
      {listOfUrls}
```
Your "li" tags are already being rendered by "listOfUrls" on line 7. No need to repeat them. Browser notifies you of this with:
> Warning: validateDOMNesting(...): ```<li>``` cannot appear as a descendant of ```<li>```...

File: prelab_src/components/Dogs.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Dog from './Dog';
+import tree1 from '../assets/tree1.jpg';
+import tree2 from '../assets/tree2.jpg';
+
+
+function Dogs({ dogs }) {
+  const listOfDogs = dogs.map(name => {
+    // const listOfDogs = ['rover', 'spot', 'bingo'].map(name => {
```
Comment (grayson073) [inline prelab_src/components/Dogs.js]:
```suggestion
```

File: prelab_src/components/Dogs.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    // li.textContent = name;
```
Comment (grayson073) [inline prelab_src/components/Dogs.js]:
```suggestion
```

File: src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './Header.css';
+
+function Header() {
+  console.log(styles);
```
Comment (grayson073) [inline src/components/Header.js]:
```suggestion
```
Don't forget to do a quick sweep for console logs, dead code, etc.

**Comment by carmenvramos** [comment]
Completed suggested changes.

**Pull request merged**